docs/source/keywords.rst

```diff
@@ -734,6 +734,11 @@ Kokkos will attempt to autodetect the architecture flag at configuration time.
 Intel GPUs
 ~~~~~~~~~~
 
+If the SYCL backend is enabled and no GPU architecture is specified,
+Kokkos will use Just-In-Time (JIT) compilation and select the first visible device for the default execution space instance.
+There is no restriction to a particular SYCL device type.
+Thus, this is the only option to target a CPU with the SYCL backend (which is experimental, untested, and not optimized for).
```

Comment:
Does that imply we define `Kokkos_ARCH_INTEL_GEN=ON` if no Intel GPU arch is enabled or is it more subtle than that?

Reply:
No, we can't assume that we are building for an Intel GPU so we don't define enable any extra `Kokkos_ARCH*` options.